compiler/src/iree/compiler/Codegen/Passes.h

```diff
@@ -275,6 +283,13 @@ createRematerializeParallelOpsPass();
 std::unique_ptr<OperationPass<func::FuncOp>>
 createInstrumentMemoryAccessesPass();
 
+/// Returns the op that contains lowering config. Checks whether the provided op
```

Comment:
This shouldnt live here. #13591 splits the `Passes.h` . We should move it in there.... 

I'd also want to try to not use `rootOperation` if possible. You are only trying to get the lowering config right? You might as well return that?

Reply:
Should we move it to `LLVMCPU/Utils` ?

Reply:
I can revisit these but IIRC there was no other way. We can move it to LLVMCPU/Utils, for sure.